edi_core_oca/models/edi_exchange_type.py

```diff
@@ -88,6 +84,23 @@ class EDIExchangeType(models.Model):
         comodel_name="edi.exchange.type",
         compute="_compute_ack_for_type_ids",
     )
+    receive_model_id = fields.Many2one(
```

Comment:
```suggestion
    receive_model_id = fields.Many2one(string="Receiver"
```
and so on for all the fields.